reactors/1D_packbed.ipynb

```diff
@@ -0,0 +1,519 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.        permeance = mem_perm/thickness               # permeance of the membrane (kmol*m2/s/Pa)
</pre><p>This looks like a constant, so you can pull it out of this function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Cantera/cantera-jupyter/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Corrected.</p>